src/backend/aspen/database/models/public_repository_metadata.py

```diff
@@ -0,0 +1,40 @@
+from sqlalchemy import Column, DateTime, ForeignKey, Integer, String, UniqueConstraint
+from sqlalchemy.orm import relationship
+
+from aspen.database.models.base import idbase
+from aspen.database.models.mixins import DictMixin
+from aspen.database.models.pathogens import Pathogen
+from aspen.database.models.public_repositories import PublicRepository
+
+
+class PublicRepositoryMetadata(idbase, DictMixin):  # type: ignore
```

Comment:
in general i'm a little confused on this table... so for a particular pathogen and public_repository we are expecting these fields? (strain, clade, lineage, isl, etc)

Reply:
we'll continue this discussion in above thread